docs/HPCCSystemAdmin/SA-Mods/SysAdminConfigMod.xml

```diff
@@ -370,4 +369,283 @@ sudo -u hpcc cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/envi
       </sect2>
     </sect1>
   </chapter>
+
+  <chapter id="Advance-SysAdmin-Topic-Chapter">
+    <title>Advanced Systems Administrator Topics</title>
+
+    <para>This chapter contains information about various Systems
+    Administrators topics.</para>
+
+    <para>This is intended as an includable module, where the sections that
```

Comment:
This looks like a comment to yourself. Is it intended to be included in published doc?